packages/core/client/src/schema-component/antd/action/hooks.ts

```diff
@@ -59,3 +59,163 @@ export const useLinkageAction = () => {
   const isAction = ['Action.Link', 'Action'].includes(fieldSchema['x-component']);
   return isAction && isRecordAction;
 };
+
+export const useCustomRequestSchema = () => {
```

Comment:
```suggestion
export const getCustomRequestSchema = () => {
```
建议不要命名成一个 `hook` 因为该函数只是简单的返回一个 `schema`。